service/matching/physical_task_queue_manager_test.go

```diff
@@ -94,6 +100,54 @@ func withIDBlockAllocator(ibl idBlockAllocator) taskQueueManagerOpt {
 	}
 }
 
+// addTasks is a helper which adds numberOfTasks to a taskTracker
```

Comment:
The unit test has been moved to this location instead of `backlog_manager.test`. While placing it inside backlog manager makes more sense, we do not need a mocked `pqMgr` since we really on active pollers changing the tasks being read from the buffer. `backlog_manager.test` consists of tests where the `pqMgr` is mocked and was causing the test logic to be faulty.